### PR TITLE
[AN-4880] Change verification warning when user is self

### PR DESCRIPTION
**Pull request by JPonte**
#### APK
[Download build #8383](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8383/artifact/build/artifact/wire-dev-PR553-8383.apk)
[Download build #8384](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8384/artifact/build/artifact/wire-dev-PR553-8384.apk)
[Download build #8403](http://192.168.10.18:8080/job/Pull%20Request%20Builder/8403/artifact/build/artifact/wire-dev-PR553-8403.apk)

### CODE REVIEW

File: app/src/main/java/com/waz/zclient/pages/main/conversation/ConversationFragment.java [hunk 1]
```diff
@@ -1633,6 +1648,25 @@ private void onErrorCanNotSentMessageToUnverifiedConversation(final ErrorsList.E
                 tmpUnverifiedDevices++;
             }
         }
+
+        if (self != null && self.getUser().getVerified() != Verification.VERIFIED) {
+            if (userCount == 0) {
```
Comment (mykola-mokhnach) [inline app/src/main/java/com/waz/zclient/pages/main/conversation/ConversationFragment.java]: nit: onlySelfChanged = userCount == 0;

**Comment by xuanzhaopeng** [comment]
Steps:

1. Have user A and user B both on Android
2. User A and user B verify all their devices in Settings
3. User A and user B verify all devices in 1:1 conversation details to make it verified
4. User B adds a new device
5. User B sends a message to user A

6. B see a popup : "You add a new device" (sth like that),  ask me cancel or send anyway. => Correct behaviour

7. B tap on "Send anyway". 

However A received B's message, A also see a popup, it says "B use a new device",  and ask A "Send anyway" or cancel.  In fact A send nothing in this moment, just receive B's message.  

Correct behaivor :
A should see a system message "B started using a new device" (sth like that),  before the B's new message

**Comment by mykola-mokhnach** [comment]
The issue is still reproducible. Make sure both users have more than one device while testinng it.

**Comment by xuanzhaopeng** [comment]
The content is changed from "Someone" To "You", related issue please check the ticket above

**Pull request merged**